ext/src/dll/opentelemetry_cpp.src

```diff
@@ -72,10 +72,18 @@ EXPORTS
     ?Create@ViewFactory@metrics@sdk@v1@opentelemetry@@SA?AV?$unique_ptr@VView@metrics@sdk@v1@opentelemetry@@U?$default_delete@VView@metrics@sdk@v1@opentelemetry@@@std@@@std@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@7@00W4AggregationType@2345@V?$shared_ptr@VAggregationConfig@metrics@sdk@v1@opentelemetry@@@7@@Z
     // public: static class std::unique_ptr<class opentelemetry::v1::sdk::metrics::View,struct std::default_delete<class opentelemetry::v1::sdk::metrics::View> > __cdecl opentelemetry::v1::sdk::metrics::ViewFactory::Create(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,enum opentelemetry::v1::sdk::metrics::AggregationType)
     ?Create@ViewFactory@metrics@sdk@v1@opentelemetry@@SA?AV?$unique_ptr@VView@metrics@sdk@v1@opentelemetry@@U?$default_delete@VView@metrics@sdk@v1@opentelemetry@@@std@@@std@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@7@00W4AggregationType@2345@@Z
+
+#if defined(WITH_STL)
```

Comment:
Given that the difference is about nostd::string_view, I suspect that the check should be similar to:

if OPENTELEMETRY_STL_VERSION >= 2017

Please test building both:

* WITH_STL=2014 (should use nostd::string_view)
* WITH_STL=2017 (should use std::basic_string_view)

Also, consider adding comments in this file, to explain the details ... this will be very invaluable when maintaining this.

Reply:
Sorry about that, I completed forgot that std::string_view was introduced only in C++17. Fixed this issue in the latest commit with the correct check ``#if (OPENTELEMETRY_STL_VERSION >= 2017)``, and tested with ``WITH_STL=OFF``, ``WITH_STL=CXX14`` and ``WITH_STL=CXX17``.